cmat128/doc.go

```diff
@@ -0,0 +1,98 @@
+// Generated by running
+//  go generate github.com/gonum/matrix
+// DO NOT EDIT.
+//
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// Package cmat128 provides implementations of complex128 matrix structures and
```

Comment:
How does this differ from he mat64 case which uses the same language s/complex128/float64/